### PR TITLE
Starting code cleanup and adding larger dropzone

### PR DESCRIPTION
**Pull request by SaFrMo**
Thanks for the merge on #32 ! I started working off of that and splitting the file opening functionality into a separate script and it felt like a good opportunity to work on the dropzone. Now it takes up the entire window and shows an icon on drag:

dropzone

Switching to normal cleanup and documentation rather than adding features now!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,8 +31,22 @@
...
+                    d="M5,3 l0,4 M3,5 l4,0"
+                    fill="none"
+                    stroke="#fff"
+                    stroke-width="0.5"
+                />
+            </svg>
+        </div>
         <div id="topBar">
-            <div id="drop-zone" class="topBarDiv dropZone">
+            <div class="topBarDiv dropZone">
```
Comment (bMacSwigg) [inline index.html]: If the whole area is a drop-zone now, is this class needed? From the CSS, it looks like all it does is adjust padding.

Comment (SaFrMo) [reply index.html]: That does look unnecessary - removing, thanks!

**Review by bMacSwigg — APPROVED** [review]
LGTM :)

**Comment by everestpipkin** [comment]
Thank you! This looks great.

**Pull request merged**